avaframe/com1DFA/particleTools.py

```diff
@@ -0,0 +1,640 @@
+"""
```

Comment:
I really like the commenting and shortness/overview of the functions in this file!!